ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -0,0 +1,99 @@
+            .init(title: "Today’s To-Do List", toDoItems: [
+                .init(title: "Cleaning", deadline: Date(), status: false),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: false),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: false)
+                
+            ]),
+            .init(title: "All To-Do List", toDoItems: [
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: true),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: true),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Painting", deadline: Date(), status: true)
+            ])
+        ]
+    var body: some View {
+        NavigationStack{
+            List {
+                ForEach(ToDoListView.items) { tasks in
+                    HStack(spacing: 6) {
+                        Text(tasks.title)
+                            .foregroundColor(Color(.caption))
+                            .font(.system(size: 12))
+                            .fontWeight(.medium)
+                        Text("\(tasks.toDoItems.capacity) tasks left")
+                            .foregroundColor(.gray)
+                            .font(.system(size: 12))
+                            .fontWeight(.thin)
+                    }
+                    ForEach(tasks.toDoItems) { todo in
+                        CellTask(todo: todo)
+                    }
+                }
+                Spacer()
+            }
+            .navigationTitle("To-Do List")
+            .listStyle(.plain)
+            .padding(.leading, 16)
+        }
+    }
+}
+struct CellTask: View {
+    var todo: TaskModel
+    var body: some View {
+        VStack(alignment: .leading) {
+            HStack(spacing: 30) {
+                Button( action: {
+                    print("Change status")
+                }, label: {
+                        Image(systemName: (todo.status ? "checkmark.circle" : "circle"))
+                            .frame(minHeight: 22)
+                            .clipShape(Circle())
+                            .foregroundColor(.blue)
```

Comment:
the color should be extracted as well

Reply:
But because we use listStyle modifier is plain, then if we remove this, the button's color will be black instead of blue as the UI design

Reply:
I mean the color should be defined in the Assets, a color style should be given for it, and a color style should be used instead of the system defined color

Reply:
Oh ok, I've fixed this.